### PR TITLE
Add badges to main readme page

### PR DESCRIPTION
**Pull request by beth-panx**
Badges for
- [x] license
- [x] issues
- [x] prs
- [x] watchers
- [x] forks
- [x] stars
- [x] open in vscode

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+[![GitHub license](https://img.shields.io/github/license/microsoft/Win-Student-Dev.svg)](https://github.com/microsoft/Win-Student-Dev/blob/master/LICENSE)
+[![GitHub issues](https://img.shields.io/github/issues/microsoft/Win-Student-Dev.svg)](https://GitHub.com/microsoft/Win-Student-Dev/issues/)
+[![GitHub pull-requests](https://img.shields.io/github/issues-pr/microsoft/Win-Student-Dev.svg)](https://GitHub.com/microsoft/Win-Student-Dev/pulls/)
```
Comment (aarongustafson) [inline docs/README.md:3]: Do we want to draw attention to PRs (and issues) if they have to do with our forthcoming articles?

Comment (beth-panx) [reply docs/README.md:3]: That's a great point! I can remove this.

Comment (beth-panx) [inline docs/README.md:3]:
```suggestion
```

File: docs/README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+[![GitHub license](https://img.shields.io/github/license/microsoft/Win-Student-Dev.svg)](https://github.com/microsoft/Win-Student-Dev/blob/master/LICENSE)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+[![GitHub watchers](https://img.shields.io/github/watchers/microsoft/Win-Student-Dev.svg?style=social&label=Watch&maxAge=2592000)](https://GitHub.com/microsoft/Win-Student-Dev/watchers/)
+[![GitHub forks](https://img.shields.io/github/forks/microsoft/Win-Student-Dev.svg?style=social&label=Fork&maxAge=2592000)](https://GitHub.com/microsoft/Win-Student-Dev/network/)
```
Comment (zateutsch) [inline docs/README.md:5]: Does same go for forks if the other weeks are in there?
```suggestion
```

**Pull request merged**